templates/docs/examples/patterns/hero/hero-50-50-stacked-on-medium.html

```diff
@@ -0,0 +1,35 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Hero / 50/50 / Stacked on medium{% endblock %}
+{% block standalone_css %}patterns_hero{% endblock %}
+
+{% set is_paper = True %}
+{% block content %}
+
+<section class="p-section--hero">
+  <div class="row--50-50-on-large">
+    <div class="col">
+      <div class="p-section--shallow">
+        <h1>H1 - ideally one line, up to two</h1>
+        <h2>H2 placeholder - aim for one line, 2 is acceptable, more - use a paragraph</h2>
+      </div>
+      <div class="p-section--shallow">
+        <p>Generally, the height of the right hand side of a 50/50 split should contain more content than the left
+          hand side.</p>
+      </div>
+      <div class="p-cta-block">
+        <a href="#" class="p-button--positive">Learn more</a>
+        <a href="#">Contact us ›</a>
+      </div>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <div class="p-image-container--16-9">
```

Comment:
I think all images need `is-cover` (if they are photograph), otherwise they don't respect aspect ratio and end up having weird spacing on sides.

```suggestion
        <div class="p-image-container--16-9 is-cover">
```